brightwind/transform/transform.py

```diff
@@ -1598,10 +1597,10 @@ def _check_vertical_profiler_properties_not_overlap(meas_station_data_models, df
             # Check if date ranges overlap
             overlap = (range1['from'] < range2['to'] and range2['from'] < range1['to'])
             
-            # If ranges overlap and at least one has non-zero offset, raise error
```

Comment:
why on line 1583 device orientation if None is set to 0? shouldn't this be left as None?